priv/repo/migrations/20180421181224_add_status_to_speakers.exs

```diff
@@ -0,0 +1,9 @@
+defmodule CbusElixir.Repo.Migrations.AddStatusToSpeakers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:speakers) do
+      add :status, :string
```

Comment:
You're going to have to remove the `email` and `name` field which got put on `Users`